deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -1576,3 +1576,18 @@ Generate list of remoteWrite endpoints for telegraf configuration
 {{- $endpoints := sortAlpha $endpoints -}}
 {{ $endpoints | join ",\n" }}
 {{- end -}}
+
+{{/*
+Return the log format for the Sumologic exporter for container logs
+
+'{{ include "metric.endpoints" . }}'
+*/}}
+{{- define "logs.otelcol.container.exporter.format" -}}
+{{- if eq .Values.sumologic.logs.container.format "json" -}}
+{{- "json" -}}
+{{- else if eq .Values.sumologic.logs.container.format "text" -}}
+{{- "text" -}}
+{{- else -}}
+{{- fail "`sumologic.logs.container.format` can only be `json` or `text`" -}}
```

Comment:
Why not allow users to set `otlp`?

Reply:
Oh in that case I think we should come up with a better name for it.

Still I believe we need to make it possible for users to send data in OTLP sources (after they have them enabled for their org), but it's a different feature than this one (although backed by the same setting in sumologic exporter).

Reply:
There's actually no difference in terms of wire format between `text` and `json` in the Sumo exporter. `text` simply ignores record attributes and just sends the body, while `json` sends a json document with keys containing record attributes and body. But fundamentally you're still sending a block of text with records separated by newlines.

Reply:
I still think we should allow `otlp` :wink: